scripts/check-security

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+# This script checks if there is plain (not redacted) log in TiKV.
+
+set -e
+
+if grep -r --color=always -n -C 1 hex::encode_upper {components,src}/ ; then
+    echo "Please follow log redaction rules. View https://github.com/tikv/tikv/pull/9250 for more information." && false
```

Comment:
Let's provide more details in this message. How about
```
To print user data into info logs or error messages, use log_wrapper::Value() instead of hex::encode_upper. The former will respect `security.redact-info-log` config and filter out user data from info log if needed. Otherwise, use `tikv_util::hex_encode_upper` to get around the lint error.
```